items.json

```diff
@@ -104342,14 +104342,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_long_pernach_v1_h0",
-    "baseId": "crpg_long_pernach_v1",
+    "id": "crpg_long_pernach_v2_h0",
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.